src/snowflake/telemetry/_internal/opentelemetry/test/metrictestutil.py

```diff
@@ -0,0 +1,100 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from opentelemetry.attributes import BoundedAttributes
+from opentelemetry.sdk.metrics.export import (
+    AggregationTemporality,
+    Gauge,
+    Metric,
+    NumberDataPoint,
+    Sum,
+)
+
+
+def _generate_metric(
```

Comment:
Does this need to be in the module that we install?

Reply:
No, I'll move this to a new module in `tests/` that only needs to be installed when running pytest.

Reply:
Please do, and when done resolve the conversation.